native/src/common/component/View.tsx

```diff
@@ -58,3 +61,27 @@ export const View: React.FC<ViewProps & { container?: boolean }> = ({
     </ViewBase>
   );
 };
+
+export const SecundaryView: React.FC<ViewProps & { container?: boolean }> = ({
```

Comment:
```suggestion
export const SecondaryView: React.FC<ViewProps & { container?: boolean }> = ({
```

Aunque el nombre y la utilidad de estas constantes me parece un poco ambigua. 

Una `View` puede tener muchas utilidades. Con el progreso del desarrollo de la app te darás cuenta que van a existir varios tipos de `View`. 

Por lo tanto, utilicemos sólo un componente `View` sin clasificarlo por primario o secundario.